dimod/discrete/discrete_quadratic_model.py

```diff
@@ -587,20 +608,30 @@ def _to_file_numpy(self, file, compress):
         file.write(b'    ')  # will be replaced by the length
         start = file.tell()
 
-        vectors = self.to_numpy_vectors()
+        vectors = self.to_numpy_vectors(return_offset=save_offset)
 
         if compress:
             save = np.savez_compressed
         else:
             save = np.savez
 
-        save(file,
-             case_starts=vectors.case_starts,
-             linear_biases=vectors.linear_biases,
-             quadratic_row_indices=vectors.quadratic.row_indices,
-             quadratic_col_indices=vectors.quadratic.col_indices,
-             quadratic_biases=vectors.quadratic.biases,
-             )
+        if save_offset:
```

Comment:
probably a bit cleaner to make
```
data = dict(case_starts=....)
if save_offset:
    data.update(offset=vectors.offset)
save(file, **data)
```

Reply:
I guess the above doesn't apply if we do away with `save_offset`